src/register/RegistrationPage.jsx

```diff
@@ -29,10 +28,12 @@ import { getBackendValidations, isFormValid, prepareRegistrationPayload } from '
 import messages from './messages';
 import { EmailField, NameField, UsernameField } from './RegistrationFields';
 import {
-  InstitutionLogistration, PasswordField, RedirectLogistration, ThirdPartyAuthAlert,
+  InstitutionLogistration,
+  PasswordField, RedirectLogistration, ThirdPartyAuthAlert,
 } from '../common-components';
 import { getThirdPartyAuthContext as getRegistrationDataFromBackend } from '../common-components/data/actions';
 import EnterpriseSSO from '../common-components/EnterpriseSSO';
+import ThirdPartyAuth from '../common-components/ThirdPartyAuth';
```

Comment:
these two should not be imported separate. should be Added  with other imports of common components.

```
  EnterpriseSSO  
  InstitutionLogistration,
  PasswordField, 
  RedirectLogistration,
  ThirdPartyAuth, 
  ThirdPartyAuthAlert,
} from '../common-components';

Reply:
actually if we add the ThirdPartyAuth in common components's index file it creates cycle dependency let me see if we can break the dependency

Reply:
```import {
  RenderInstitutionButton,
  SocialAuthProviders,
} from './index';``` this import in ThirdPartyAuth component needs to change we need to directly import RenderInstitutionButton and SocialAuthProviders directly instead of index file then it would solve the issue do you want me to do it?